### PR TITLE
Updated transcoding post

### PR DESCRIPTION
**Pull request by cpitkin**

### CODE REVIEW

File: content/blog/automated-video-transcoding-with-openfaas.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+The companion uploader executable can be found [here](https://github.com/cpitkin/video-upload/)
+
+# Architecture
+
+![architecuture diagram](/img/tp.png)
+
+## Considerations
+
+- Transcoding is a very CPU intensive task. Regardless of how you configure the setup, it is going to take significant CPU and disk i/o resources. That being said you can run this setup on pretty much any hardware but the more resources you give the transcode container the faster it will run. This setup has been tested using an Ubuntu Linux box with a 6 core 3.5 GHz AMD processor.
```
Comment (alexellis) [inline content/blog/automated-video-transcoding-with-openfaas.md]: Mention how much RAM you have. If possible the AMD processor type. (Run NodeInfo..)

File: content/blog/automated-video-transcoding-with-openfaas.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+- We will be uploading raw video to one bucket (transcode) and placing the transcoded video in another bucket (complete). This means you will need enough disk space to accommodate at least your original file size*2. Ideally, for lots of files, you would want at least 500GB of free space.
```
Comment (alexellis) [inline content/blog/automated-video-transcoding-with-openfaas.md]: Lots of files == 500gb? Can you be more exacting .. like for around 5 Blu-rays you'll need around X GB

File: content/blog/automated-video-transcoding-with-openfaas.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+### Setup
+
+### Prerequisites
+
+You will need a one or two Minio servers setup. In my case, I use two located on two different physical servers but you can easily do it with one.
```
Comment (alexellis) [inline content/blog/automated-video-transcoding-with-openfaas.md]: Did you introduce Minio as a concept yet? Link out to QuickStart?

File: content/blog/automated-video-transcoding-with-openfaas.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      }
+    }
+    ...
+  }
+}
+```
+
+**NOTE:** If you're using the uploader executable mentioned above then the PutObject event trigger will be set for you. If you will need to manually set a PutObject event trigger for the webhook on the transcode bucket.
+
+## OpenFaaS Functions (in order)
```
Comment (alexellis) [inline content/blog/automated-video-transcoding-with-openfaas.md:74]: "A picture speaks 1000 words" - put one together with draw.io or similar.

File: content/blog/automated-video-transcoding-with-openfaas.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+You will need a one or two Minio servers setup. In my case, I use two located on two different physical servers but you can easily do it with one.
+
+Buckets:
+
+- transcode
+- complete
+- movies
+- tv
+
+#### Minio config
```
Comment (alexellis) [inline content/blog/automated-video-transcoding-with-openfaas.md:41]: Are you running Minio in K8s / Docker / on the host?

Comment (cpitkin) [reply content/blog/automated-video-transcoding-with-openfaas.md:41]: Addressed this in the latest commit.

File: content/blog/automated-video-transcoding-with-openfaas.md (hunk 4 above)
Comment (cpitkin) [reply content/blog/automated-video-transcoding-with-openfaas.md:74]: There is an architecture reference on line 18

**Pull request merged**